### DISCUSSION TITLE
xtts  v2

### DISCUSSION THREAD

**Discussion by windkwbs** [open-webui/open-webui#3344] [General]
![微信截图_20240621112225](https://github.com/open-webui/open-webui/assets/129468439/4de1a1f6-3d2d-45eb-9728-d5c6c7ff55d8)
![微信截图_20240621112317](https://github.com/open-webui/open-webui/assets/129468439/456a1256-1076-4525-bd92-b9cd1b1236c5)

**Comment by justinh-rahb** [comment]
> `http://0.0.0.0:8020`

In all the time you spent drawing red circles and arrows you didn't think to maybe try asking an LLM about why this very obvious error is blocking your path? Or even baidu search engine? I think either would have been quicker.

**Reply by windkwbs** [reply]
http://localhost:8020 It's not possible either
